src/getNOAA.R

```diff
@@ -12,21 +12,22 @@
 #' @return An sf object of weather stations within given spatial boundary and attributes
 getNOAA <-
   function(aoi,
-           dist = 0,
+           park,
```

Comment:
will review if adding park as an argument is necessary based on how AOIs for this project are developed.